spec/component_guide/component_example_accessibility_testing_spec.rb

```diff
@@ -11,19 +11,14 @@
     expect(page).to have_selector('.js-test-a11y-success.js-test-a11y-finished')
   end
 
-  it 'throws JavaScript errors if a component has an accessibility issue' do
-    expect { visit '/component-guide/test-component-with-a11y-issue' }
-      .to raise_error(Capybara::Poltergeist::JavascriptError)
-  end
-
-  it 'shows accessibility violations on the page' do
-    expect { visit '/component-guide/test-component-with-a11y-issue' }
-      .to raise_error(Capybara::Poltergeist::JavascriptError)
-
+  it 'shows accessibility violations on the page and through browser console' do
+    visit '/component-guide/test-component-with-a11y-issue'
     expect(page).to have_selector('.js-test-a11y-failed.js-test-a11y-finished')
 
+    expect(page.driver.browser.manage.logs.get(:browser).map { |e| e.message if e.message.match(/Accessibility issues/) }).not_to be_empty
+
     selector_with_error = page.first('.selector').text
-    expect(page).to have_selector(selector_with_error)
+    expect(page).to have_selector(selector_with_error, visible: false)
```

Comment:
Why is this not visible?

Reply:
The element we're selecting is `<img src="">`.  It seems Poltergeist/PhantomJS saw this as visible, whereas Selenium/Chrome doesn't.

Reply:
May be worth adding a comment, e.g.

```ruby
# The deliberately inaccessible HTML `<img src="">` is invisible, so needs `visible: false`
expect(page).to have_selector(selector_with_error, visible: false)
```